### DISCUSSION TITLE
[i'm noob] unable to change flashmq.conf when running on docker

### DISCUSSION THREAD

**Discussion by itweb62** [halfgaar/FlashMQ#40] [General]
i don't know how to change the conf file when i try to login with allow anonymous yes i got access denied.. 

Thank you

**Comment by halfgaar** [comment]
Did you specify a volume for the internal container location `/etc`? Did you edit the config file there?

Also, to reload the config in the container, you'll have to do something like `docker exec <container> killall -SIGHUP flashmq`.

**Comment by itweb62** [comment]
Thank you i have find this.

Best regards

**Comment by itweb62** [comment]
i have a other question..
Do you think a redis plugin can be achieved by myself for vertical scaling FlashMQ on a cluster configuration ? 

(i want this just for "presence" online status or send message for tchat app) 
i don't know how to create shared object (.so) of h-redis lib https://github.com/redis/hiredis
i know how to create .h file.

**Reply by halfgaar** [reply]
You mean horizontal scaling? No, that can't be done with the plugin interface.

What is your need for clustering? On it's own, it can handle millions of clients.

**Reply by itweb62** [reply]
No vertical for loosing less money. 
i don't have money for 64 dedicated core server.  ahah

**Reply by halfgaar** [reply]
But you do mean multiple servers? That would be horizontal.

But what is your workload and available server? The idea of FlashMQ is that it can already do so much on a single server that many use cases don't need clustering.

**Reply by itweb62** [reply]
Yes multiple servers , i would only connect 500k user / 1 millions over 10 server with 4 CPU  / 6 CPU core with load balancing. 
the tchat app gonna grow that why i need clustering and Redis in-memory DB

**Reply by itweb62** [reply]
500k/1 millions user can be achieved with how many CPU core in 1 instance ? 16 CPU no ?

**Reply by halfgaar** [reply]
As you can see [here](https://www.youtube.com/watch?v=ejS0j1HjVdk&t=305s) at that time index, loading 750k users on a four core is easy. Idling users hardly take CPU. Then, it's still able to handle 1 million messages per second.

**Reply by itweb62** [reply]
Thank you a lot
